### PR TITLE
Deleted the flattened contract and Added metakeep-lambda sdk.

### PR DESCRIPTION
**Pull request by aditya-passbird**

### CODE REVIEW

File: lambda/voting/scripts/create.js [hunk 1]
```diff
@@ -30,32 +29,12 @@ async function main() {
   );
   const developerAddress = await getDeveloperWallet();
 
-  const requestBody = {
```
Comment (rohan-passbird) [inline lambda/voting/scripts/create.js:33]: Are the url and headers above needed?

File: lambda/lambdaUtils.mjs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  const result = await fetch(url, options);
+  const resultJson = await result.json();
+
+  console.log(resultJson);
+
+  if (!result.ok) {
+    console.log(
+      "Error while invoking method. HTTP status code: " + result.status
+    );
+    exit(1);
```
Comment (rohan-passbird) [inline lambda/lambdaUtils.mjs:39]: instead of this. let's throw an error instead. same below.

**Review by rohan-passbird — DISMISSED** [review]
👍

File: lambda/voting/scripts/create.js (hunk 1 above)
Comment (aditya-passbird) [reply lambda/voting/scripts/create.js:33]: removed

File: lambda/lambdaUtils.mjs (hunk 2 above)
Comment (aditya-passbird) [reply lambda/lambdaUtils.mjs]: done

File: lambda/lambdaUtils.mjs [hunk 3]
```diff
@@ -0,0 +1,85 @@
+//Helper functions for the MetaKeep APIs
+
+import fetch from "node-fetch";
+import { exit } from "process";
+
+// Invokes the lambda function.
+export default async function invoke(functionName, functionArgs, reason) {
```
Comment (rohan-passbird) [inline lambda/lambdaUtils.mjs:7]:
```suggestion
export const invoke = async(functionName, functionArgs, reason) {
```

For consistency with others

**Review by rohan-passbird — DISMISSED** [review]
👍

**Review by rohan-passbird — APPROVED** [review]
👍

**Pull request merged**